### DISCUSSION TITLE
Responsive Web Design

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#48] [Q&A] [answered]
What is Responsive Web Design (RWD) in HTML and CSS?

**Comment by Bunny77K9** [comment, marked as answer]
Responsive Web Design is a concept that is used to create web pages that can scale across multiple resolutions without any loss of information or screen tearing.

It automatically adjusts the structure of the web page based on the device it is viewed on to provide optimal viewing experience.
